packages/@lwc/engine-server/src/__tests__/fixtures/attribute-dynamic-with-scoped-css/modules/x/attribute-dynamic-with-scoped-css/attribute-dynamic-with-scoped-css.js

```diff
@@ -0,0 +1,5 @@
+import { LightningElement } from 'lwc';
```

Comment:
The brokenness was in the validation client-side. However, I thought it worthwhile adding coverage for the dynamic class here, since I didn't see it in existing `engine-server` tests.